### ISSUE TITLE
Not working on RoS ver Mar 14th

### ISSUE THREAD

**Issue by Sp-Lily** [Ashesh3/ROSUltraHack#13]
The hack doesn't work on recent version of RoS and rarely it'll crash the game.

**Comment by Allysh0w** [comment]
Maybe the offset is wrong

**Comment by Sp-Lily** [comment]
Probably, but I'm still waiting for the devs to correct it

**Comment by Allysh0w** [comment]
Strange.. I checked the offsets and it seems correct.

**Comment by nelski06** [comment]
i think the pointer is incorrect but the off sets is correct

**Comment by Sp-Lily** [comment]
Yeah it can be too. 2x I compile and test it, the hack running smooth but nothing works (No ESP Effect, etc).

**Comment by nelski06** [comment]
yeah the chams is working but the esp is not working

**Comment by Sp-Lily** [comment]
The recent update changed the address of the pointer

**Comment by nelski06** [comment]
you have new pointers ?

**Comment by Sp-Lily** [comment]
Nope, I'm not really advanced on C# (Not learning about pointer yet). We can wait for the owner of this hack to figure that out.

Is this project watched by the owner?

Btw, his post link at unknown cheats is dead. Visiting GitHub is the only way to report bugs

**Comment by Ashesh3** [comment]
The game has updated the way entity looping works.. you can still update this by referring to 

https://github.com/isysforce/SimpleESP/blob/master/SimpleBone/Form1.vb


Good luck!

**Comment by Sp-Lily** [comment]
That program still unstable. It's crashed when I launch it. 

I hope this hack still updated

**Comment by Sp-Lily** [comment]
I am sure this project will be abandoned.. well gotta find another hacking source
